Sources/SecurityRequirement.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import ObjectMapper
+
+public struct SecurityRequirement: ImmutableMappable {
+    public let name: String
+    public let scopes: [String]
+
+    public init(map: Map) throws {
+        name = map.JSON.first!.key
```

Comment:
Ah, I think if we wanted to do it this way we might have to map it to a [String: Any] first. But your new approach is good too.